astropy_bot/tests/test_autolabel.py

```diff
@@ -0,0 +1,60 @@
+# For these tests, we patch the repo and pull request handler directly rather
+# than the requests to the server, as we assume the repo and pull request
+# handlers are tested inside baldrick.
+
+from unittest.mock import MagicMock
+
+from astropy_bot.autolabel import autolabel_subpackages
+
+
+class TestAutolabel:
+
+    def setup_method(self, method):
+
+        self.files = ['CHANGES.rst', 'astropy/io/fits/convenience.py',
```

Comment:
I think it is worth adding more test methods to test different scenarios.

Reply:
Do you have any suggestions for "different scenarios"?

Reply:
Something like https://github.com/astropy/astropy-bot/pull/106#discussion_r295892342

Reply:
OK, I added one other scenario, but IMO the rest of the logic and scenarios are out of scope for this PR.